src/v/cluster/bootstrap_backend.cc

```diff
@@ -203,7 +203,14 @@ bootstrap_backend::apply(bootstrap_cluster_cmd cmd, model::offset offset) {
           [o = offset,
            m = cmd.value.recovery_state->manifest,
            b = cmd.value.recovery_state->bucket](auto& recovery_table) {
-              recovery_table.apply(o, m, b, wait_for_nodes::yes);
+              auto ec = recovery_table.apply(o, m, b, wait_for_nodes::yes);
+              if (!ec) {
```

Comment:
is this checking for success but intended to check for failure?

Reply:
oof.

![oof](https://github.com/redpanda-data/redpanda/assets/8669100/8eb68598-a3e2-4070-8fdc-cfa434f91af8)